### PR TITLE
tee/epochproch Ease error handling

### PR DESCRIPTION
**Pull request by ndzik**
Signed-off-by: Norbert Dzikowski <norbert@perun.network>

### CODE REVIEW

File: tee/prototype/epochproc.go [hunk 1]
```diff
@@ -101,7 +101,7 @@ func (e *Enclave) txRoutine(
 
 			bps, err := e.generateBalanceProofs(txEpoch)
 			if err != nil {
-				return fmt.Errorf("generating balance proofs: %w", err)
+				log.Errorf("generating balance proofs: %w", err)
```
Comment (ndzik) [inline tee/prototype/epochproc.go]:
```suggestion
				log.Errorf("txRoutine: generating balance proofs: %v", err)
```

Comment (sebastianst) [inline tee/prototype/epochproc.go]: if this errors, it's really broken, so better leave as is. (note that you shouldn't use `%w` in logging)

File: tee/prototype/epochproc.go [hunk 2]
```diff
@@ -122,7 +122,6 @@ func (e *Enclave) txRoutine(
 			err := e.applyEpochTx(txEpoch, tx)
 			if err != nil {
 				log.Errorf("txRoutine: Error applying tx: %v", err)
```
Comment (sebastianst) [inline tee/prototype/epochproc.go:125]: do `log := log.WithFields` in line 115 then we have nice context information here ;)

**Review by sebastianst — CHANGES_REQUESTED** [review]
1 nicety ;)

**Review by sebastianst — APPROVED** [review]

**Pull request merged**